src/ECharts/Theme.purs

```diff
@@ -1,58 +1,45 @@
 module ECharts.Theme
   ( Theme(..)
+  , BuiltInTheme
+  , builtInThemeName
   , dark
   , infographic
   , macarons
   , roma
   , shine
   , vintage
-  , BuiltInTheme(..)
-  , parseBuiltInTheme
-  , builtInToTheme
   ) where
 
-import Prelude (($), (<>))
 import Data.Either (Either(..))
 import Data.Foreign (Foreign)
 
-foreign import forceExport ∷ Foreign
+type Theme = Either BuiltInTheme Foreign
 
-data Theme = ByName String | FromObject Foreign
-data BuiltInTheme = Infographic | Macarons | Roma | Shine | Vintage | Dark
+foreign import data BuiltInTheme :: Type
 
-parseBuiltInTheme ∷ String → Either String BuiltInTheme
```

Comment:
We use `parseBuiltInTheme <#> builtInToTheme` to get Theme from string. 

As we don't have `BuiltInTheme` as sum type  I think we can just change `parseBuiltInTheme` so that it returns `Either String Theme`